modules/solid_mechanics/include/timeintegrators/CentralDifferenceDirect.h

```diff
@@ -38,13 +37,9 @@ class CentralDifferenceDirect : public ExplicitTimeIntegrator
 
   virtual bool performExplicitSolve(SparseMatrix<Number> & mass_matrix) override;
 
-  void computeADTimeDerivatives(ADReal & ad_u_dot,
-                                const dof_id_type & dof,
-                                ADReal & ad_u_dotdot) const override
+  void computeADTimeDerivatives(ADReal &, const dof_id_type &, ADReal &) const override
   {
-    ad_u_dotdot.value();
-    ad_u_dot.value();
-    return mooseError("On %d - NOT SUPPORTED", dof);
+    return mooseError("NOT SUPPORTED");
```

Comment:
no need to `return` here

Reply:
[736c3f4](https://github.com/idaholab/moose/pull/28175/commits/736c3f45bea7fbcebac66ba88f30a5be7e0b0c79)